requirements/edx/github.txt

```diff
@@ -47,7 +47,8 @@
 
 # Third-party:
 git+https://github.com/cyberdelia/django-pipeline.git@1.5.3#egg=django-pipeline==1.5.3
-git+https://github.com/edx/django-wiki.git@v0.0.9#egg=django-wiki==0.0.9
+
+git+https://github.com/raccoongang/django-wiki.git@v0.0.9.1#egg=django-wiki==0.0.9.1
```

Comment:
`v0.0.9.1` :)
Change please

Reply:
![screenshot 2019-01-29 at 13 40 37](https://user-images.githubusercontent.com/40295846/51905994-83ea1480-23cb-11e9-8ed3-02734610cbde.png)
change to what ?:)